.travis.yml

```diff
@@ -64,7 +64,8 @@ jobs:
         - ./scripts/gen-s-parser.py | diff src/gen-s-parser.inc -
         - BUILD_SUBDIR=${BUILD_SUBDIR:-.}
         - mkdir -p ${BUILD_SUBDIR} && cd ${BUILD_SUBDIR}
-        - cmake ${TRAVIS_BUILD_DIR} -DCMAKE_C_FLAGS="$COMPILER_FLAGS" -DCMAKE_CXX_FLAGS="$COMPILER_FLAGS" -DCMAKE_INSTALL_PREFIX=install
+        - cmake ${TRAVIS_BUILD_DIR} -DCMAKE_C_FLAGS="$COMPILER_FLAGS" -DCMAKE_CXX_FLAGS="$COMPILER_FLAGS" -DCMAKE_INSTALL_PREFIX=install -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
```

Comment:
where are those compile commands used?

Reply:
`-DCMAKE_EXPORT_COMPILE_COMMANDS=ON` adds compile_commands.json file in the build directory, which is needed by clang-tidy to run, because clang-tidy does not only check something like braces after `if` but also things that require compilation context.

Reply:
Interesting, thanks!